README.md

```diff
@@ -1 +1,77 @@
+# NextJS Custom Menu App
 
+## URLs
+
+_Note: Instructions on how to fetch menu IDs for a given API key are available in the "Multi-location support" section below._
+
+_Example menuId: 3404a063-2bbe-4473-bb1b-cac3c7e1d14e_
+
+**Localhost**
+
+- Main Menu Board: `http://localhost:3000?espressoAndCoffeeSubheading={espressoAndCoffeeSubheading}&espressoAndCoffeeBottomText={espressoAndCoffeeBottomText}&footnote={footnote}&menu={menuId}`
+
+**Production**
+
+<!-- - Main Menu Board: https://custom-menu-app-1.netlify.app?espressoAndCoffeeSubheading=<espressoAndCoffeeSubheading>&espressoAndCoffeeBottomText=<espressoAndCoffeeBottomText>&
+  footnote=<footnote>&menuId={menuId} -->
+
+## Prerequisities
+
+- Node
+- Yarn
+
+## Development
+
+1. Run `cp .env.sample .env.local` and populate env vars
+
+2. Run the development server
+
+```bash
+yarn dev
+```
```

Comment:
Nice 💯 

I was a bit lazy with the READMEs for the other [sample apps](https://github.com/mirainc/simple-weather-app). Mind making a Trello card (or two) to improve the READMEs of the other sample apps with what you've done here?

Reply:
Np! I've updated the README for the Wholefoods app already. Thanks again for porting those over 🙏 

https://trello.com/c/B1rRV71P/4532-sample-apps-update-readme-for-simple-weather-app